croncert-config.yml

```diff
@@ -699,6 +699,59 @@ scrapers:
           selector: div.row.sk_indent.gig.xs-card > div.col-xs-12.col-sm-8.col-md-7.card > div.hidden-xs > div.tickets > a.boxoffice.ticket_link.information
           attr: href
 
+  - name: DClub
+    url: https://dclub.ch/agenda/
+    item: div.dclub-agenda > div.date_container > div.events_container > div.event
+    fields:
+      - name: "type"
+        value: "concert"
+      - name: "city"
+        value: "Lausanne"
+      - name: "location"
+        value: "D! Club"
+      - name: "sourceUrl"
+        value: "https://dclub.ch/agenda/"
+      - name: style
+        type: text
+        location:
+          selector: div.info > div.details > p.style
+      - name: url
+        type: url
+        location:
+          selector: a.main
+          attr: href
+      - name: title
+        type: text
+        location:
+          selector: div.header > .artist
+          entire_subtree: true
+      - name: date
+        type: date
+        components:
+          - covers:
+              day: true
+              month: true
+              year: true
+            location:
+              selector: a.main
+              attr: href
+              regex_extract:
```

Comment:
You probably meant to do sth like this to extract the date from the url. Like this it works.

Reply:
Extracting the date from the main page also gives us the advantage of being able to include _all_ events even if there are multiple events on a single day.

Reply:
I tried that, but for some reason it didn't seem to work.

Reply:
hmmm that's strange

Reply:
Probably I missed some tiny thing. It looked like I wasn't getting a regex match in the URL with [0-9]{8}. Maybe I should have used a more precise expression.